core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/chain/UpgradeDistroxFlowEventChainFactory.java

```diff
@@ -41,7 +41,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(UpgradeDistroxFlowEventChainFactory.class);
 
-    @Value("${cb.upgrade.validation.enabled:true}")
+    @Value("${cb.upgrade.validation.distrox.enabled:false}")
```

Comment:
Can you please remove the default value from here and move to app.yml? You might need to add it to multiple app.yml-s.